pkg/psmdb/backup/job.go

```diff
@@ -1,17 +1,23 @@
 package backup
 
 import (
+	"fmt"
+
 	batchv1 "k8s.io/api/batch/v1"
 	batchv1b "k8s.io/api/batch/v1beta1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	api "github.com/percona/percona-server-mongodb-operator/pkg/apis/psmdb/v1"
+	"github.com/percona/percona-server-mongodb-operator/pkg/psmdb"
 )
 
-func BackupCronJob(backup *api.BackupTaskSpec, crName, namespace string, backupSpec api.BackupSpec, imagePullSecrets []corev1.LocalObjectReference) *batchv1b.CronJob {
+func BackupCronJob(backup *api.BackupTaskSpec, crName, namespace string, backupSpec api.BackupSpec, imagePullSecrets []corev1.LocalObjectReference) (batchv1b.CronJob, error) {
 	jobName := crName + "-backup-" + backup.Name
-
+	resources, err := psmdb.CreateResources(backupSpec.Resources)
+	if err != nil {
+		return batchv1b.CronJob{}, fmt.Errorf("cannot parse Backup resources: %w", err)
```

Comment:
please use github.com/pkg/errors package

Reply:
@egegunes  it was done 44f5c08